ironfish/src/rpc/routes/wallet/importAccount.ts

```diff
@@ -54,7 +54,7 @@ routes.register<typeof ImportAccountRequestSchema, ImportResponse>(
       ...accountImport,
     })
 
-    if (request.data.rescan) {
+    if (request.data.rescan && node.wallet.nodeClient) {
```

Comment:
I was wondering if we should check if this client is connected, but there is retry logic so checking non-null seems good.